test/blocks/global-navigation/utilities/utilities.test.js

```diff
@@ -188,14 +188,29 @@ describe('global navigation utilities', () => {
         path: '/federal/media.png',
         locale: '/ch_de',
       });
-      federatePictureSources(template);
+      federatePictureSources({ section: template });
       verifyImageTemplate({
         host: baseHost,
         path: '/federal/media.png',
         locale: '/ch_de',
         template,
       });
     });
+
+    it('should allow to force picture federation to /federal/media.png', async () => {
+      const template = getImageTemplate({
+        host: '.',
+        path: '/media.png',
+        locale: '',
+      });
+      federatePictureSources({ section: template, forceFederate: true });
+      verifyImageTemplate({
+        host: baseHost,
+        path: '/federal/media.png',
+        locale: '',
```

Comment:
I think this can be removed as the `verifyImageTemplate` function has `''` as the initial value for `locale`

Reply:
Great feedback, done! Also on older tests